### DISCUSSION TITLE
Blog ideas for Janssen Server

### DISCUSSION THREAD

**Discussion by ossdhaval** [JanssenProject/jans#2584] [Ideas]
Janssen community can use this discussion to propose blog topics. 

Blogs can aid product documentation by having focused content. Blogs about a specific feature, tip-and-tricks, how-to or any other topic where the community feels the need for content can be requested here. Community members with expertise and in-depth knowledge in relevant areas will pick up these topics and write blogs.

**Comment by ossdhaval** [comment]
Feature blog: Sector Identifiers
Feature blog: Pairwise Identifiers

**Comment by BainMcKay** [comment]
On a priority basis, we need an Admin UI and we need an export/import from GLUU 4 to Jansen., so there is a one-to-one mapping. A single click migration would save a ton of time, and frustration working through ambiguous documentation where details are understated or assumed.

Our customers use GLUU Admin UI to add users and do their own configurations. So in addition to deployment Admin UI, we need user Admin-OperationsUI CRUD, so users can be self-sufficient

**Reply by nynymike** [reply]
Gluu makes a commercial Admin UI in the Flex product. Any contributor can start an Admin UI project in their own Github repo and propose a contribution to the Janssen Project.  This Discussion forum is not the right place to make feature requests to Gluu 4. You can try GluuFederation/oxTrust for that.

**Comment by BainMcKay** [comment]
Sorry, I was suggesting a Janssen Admin UI that can hydrate from their GLUU counterpart, in migration fashion. Single button click. Point to the source. Click. One and done. 

Same with Janssen user Admin UI. Move to Janssen should be relatively transparent. Once in Janssen, users have an HTML User Admin screen to manage users (CRUD).
